bin/ci

```diff
@@ -98,7 +98,7 @@ prepare_build() {
 
 verify_version() {
   # If building a tag, check it matches with file
-  FILE_VERSION=`cat VERSION`
+  FILE_VERSION=`cat ./src/VERSION`
```

Comment:
```suggestion
  FILE_VERSION="$(cat ./src/VERSION)"
```

Reply:
Why?

Reply:
I mean, it was using backticks before and it was working fine. Why risk changing it to something that apparently works the same but might break in some other subtle ways?

Reply:
Even if it is a positive change, I would keep it separate from this PR as unrelated.

Reply:
Ok, commit removed.